desktop/mac/ChainCore/ChainCore.swift

```diff
@@ -15,6 +15,10 @@ class ChainCore: NSObject {
         return "postgres://localhost:\(dbPort)/core?sslmode=disable"
     }
 
+    var homeDir: String {
+        return NSProcessInfo().environment["HOME"] + "/Library/Application Support/Chain Core"
```

Comment:
I like this, but we've also got https://developer.apple.com/reference/foundation/1413045-nshomedirectory